data/presets/presets/barrier/kerb.json

```diff
@@ -0,0 +1,14 @@
+{
+    "icon": "wheelchair",
+    "fields": [
+        "access"
```

Comment:
I don't think "access" field makes sense here. 
You could add "kerb" and "tactile_paving" in its place, since we already have fields set up for those tags..

Reply:
sure thing!